src/index.tsx

```diff
@@ -24,6 +24,13 @@ initShortcuts()
 
 const storage: Storage = window.localStorage
 
+let languageCode = configStore.get('language')
+
+if (!languageCode) {
+  languageCode = storage.getItem('language') || 'en'
+  configStore.set('language', languageCode)
+}
```

Comment:
we were only storing the apps language in the browser's local storage, but that's inaccessible from the backend for the launch command

this block will backfill the user's language in the config store, it's kinda ugly but I couldn't find a better way to do this

another option is to NOT backfill the config value but I imagine users will think it's a bug if the OS and Heroic has different languages